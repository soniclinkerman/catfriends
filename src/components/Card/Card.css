.img-size{
    width:200px;
    height:200px;
}

.border{
    border: 1px solid #e0dbdb;
}