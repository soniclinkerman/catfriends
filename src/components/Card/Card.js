import React, {Fragment} from "react";
import "./Card.css"

const Card = ({id,name,email}) => {
    return(
        <Fragment>
            <div className="tc grow br3 pa3 ma2 dib bw2 shadow-5">
                <img className="img-size" src={`https://robohash.org/${id}?set=set4`} alt="kitten"/>
                <div>
                    <h2>{name}</h2>
                    <p>{email}</p>
                </div>
            </div>
        </Fragment>
    )
}

export default Card