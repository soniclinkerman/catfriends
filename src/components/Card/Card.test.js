import React from "react";
import "../../setupTest";
import { shallow, mount, render } from "enzyme";
import Card from "./Card";


it("should render Card Component",() => {
    expect(shallow(<Card/>)).toMatchSnapshot();

})
