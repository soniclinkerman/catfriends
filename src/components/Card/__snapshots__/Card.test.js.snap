// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Card Component 1`] = `ShallowWrapper {}`;
